packages/back-end/src/models/WebhookModel.ts

```diff
@@ -39,7 +41,18 @@ type WebhookDocument = mongoose.Document & WebhookInterface;
 const WebhookModel = mongoose.model<WebhookInterface>("Webhook", webhookSchema);
 
 function toInterface(doc: WebhookDocument): WebhookInterface {
-  return omit(doc.toJSON<WebhookDocument>(), ["__v", "_id"]);
+  const oldDoc = doc.toJSON<WebhookDocument>();
+  const newDoc = omit(oldDoc, ["__v", "_id", "sendPayload"]);
+  if (!oldDoc.payloadFormat) {
+    if (oldDoc.httpMethod === "GET") {
+      newDoc.payloadFormat = "none";
+    } else if (oldDoc.sendPayload) {
+      newDoc.payloadFormat = "standard";
+    } else {
+      newDoc.payloadFormat = "standard-no-payload";
+    }
```

Comment:
I would put this into its own stand-alone method so that it doesn't get erased by mistake. Also, could we save the migrated payload after applying it?

Reply:
sure we can refactor. as for saving the model on migration, we typically don't do that unless the conversion is expensive